### PR TITLE
Added initial UI: sidebar, menu, etc.

### PR DESCRIPTION
**Pull request by waterwo2**
Added in initial ui components, sidebar, small css, header. Image inclusion and functionality not included in this story.
Jira Story: https://tim-kaufenberg.atlassian.net/browse/ARC-55

### CODE REVIEW

**Comment by coughenberg** [comment]
Please add a description to this pr of what you edited, the Jira story this is referencing, and change the title to be more specific on what is being changed per https://github.com/ARC-Summer-2020/AutoRockClimber/wiki/Pull-Review-Expectations

File: src/App.tsx [hunk 1]
```diff
@@ -1,5 +1,238 @@
...
+import Typography from '@material-ui/core/Typography';
+import Divider from '@material-ui/core/Divider';
+import IconButton from '@material-ui/core/IconButton';
+import MenuIcon from '@material-ui/icons/Menu';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+// import ListItem from '@material-ui/core/ListItem';
+// import ListItemIcon from '@material-ui/core/ListItemIcon';
+// import ListItemText from '@material-ui/core/ListItemText';
+// import InboxIcon from '@material-ui/icons/MoveToInbox';
+// import MailIcon from '@material-ui/icons/Mail';
```
Comment (mattshoobs) [inline src/App.tsx]: Remove commented out imports that we are not using.

File: src/App.tsx [hunk 2]
```diff
@@ -1,5 +1,238 @@
...
+    width: drawerWidth,
+    flexShrink: 0,
+  },
+  drawerPaper: {
+    width: drawerWidth,
+  },
+  drawerHeader: {
+    display: 'flex',
+    alignItems: 'center',
+    padding: theme.spacing(0, 1),
```
Comment (mattshoobs) [inline src/App.tsx:57]: Fine for now, once we display an image I think we will need to increase this or come up with a different solution.

File: src/App.tsx [hunk 3]
```diff
@@ -1,5 +1,238 @@
...
+          accumsan lacus vel facilisis. Nulla posuere sollicitudin aliquam ultrices sagittis orci a.
+          <br/><br/>
+          Planning mauris now lives only once. Nulla pede ullamcorper need to soccer diameter. The cushion
+          element of the embedded dictum sagittis ac tincidunt. But unless the pool deck suspended, but the 
+          Product. Chili carrots weekend photography start. But the element Eu hate the disease. Performance 
+          skirt smile vulputate at any hardware. Website link football cartoon chili mass consumer or law 
+          enforcement need. In the makeup, Whosoever shall not nibh sed ac hendrerit gravida ac. Nor for any 
+          sem et tortor Pellentesque. They inhabit the sad old age and disease. Importantly developer homework 
+          is sad more items. Default jasmine Performance element only one deductible. Please pull Maecenas layer 
+          tanks or antioxidants. There is no set clinical care of some avenging arrows.
```
Comment (mattshoobs) [inline src/App.tsx]: Remove any old commented out code to clean up the file. We wont need the placeholder text.

File: src/App.tsx [hunk 4]
```diff
@@ -1,5 +1,238 @@
...
+          ))}
+          </List> */}
+          {/* <Divider />
+          <List>
+          {['All mail', 'Trash', 'Spam'].map((text, index) => (
+            <ListItem button key={text}>
+              <ListItemIcon>{index % 2 === 0 ? <InboxIcon /> : <MailIcon />}</ListItemIcon>
+              <ListItemText primary={text} />
+            </ListItem>
+          ))}
```
Comment (mattshoobs) [inline src/App.tsx]: Remove old commented out code

**Review by mattshoobs — CHANGES_REQUESTED** [review]
Clean up by removing old and unused commented out code then it looks good.

**Review by mattshoobs — APPROVED** [review]

**Review by skitscha — APPROVED** [review]

**Pull request merged**